### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Introduces GitHub Actions
- Adds, changes and deletes some project files

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Blaine Bublitz, Eric Schoffstall and other contributors
+Copyright (c) 2015, 2021 Blaine Bublitz, Eric Schoffstall and other contributors
```
Comment (phated) [inline LICENSE]:
```suggestion
Copyright (c) 2015-2016, 2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```

**Review by phated — APPROVED** [review]
This looks great! Thanks! We'll want to take care of #7 before publishing the new major.

**Comment by phated** [comment]
@sttk it's funny because this module doesn't need to exist anymore now that it doesn't support node 0.10 🤣 We should probably release 3.0.0 but remove it from gulp and then deprecate it.

**Comment by sttk** [comment]
Oh, sorry! I modified this without looking deeply.

**Pull request merged**